src/components/input/CInputSearch.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <b-input-group
+    class="h-100 mw-100"
+  >
+    <b-input
+      ref="value"
+      v-model="value"
+      data-test-id="input-search"
+      class="h-100 pr-0 border-light border-right-0 text-truncate bg-white"
+      type="search"
+      name="search"
+      :debounce="debounce"
+      :disabled="disabled"
+      :placeholder="placeholder"
+      :autocomplete="autocomplete"
+      @update="search($event)"
+    />
+    <b-input-group-append
+      class="bg-white border-light rounded-right append-group-border"
+    >
+      <b-button
+        v-if="showCloseButton"
+        variant="link"
+        @click="clearQuery()"
+      >
+        <font-awesome-icon
+          class="text-secondary"
+          :class="{'mt-1': showBothButtons}"
+          :icon="['fas', 'times']"
+        />
+      </b-button>
+      <b-button
+        v-if="showSearchButton"
+        variant="link"
+        :class="{'search-icon-border': showBothButtons}"
+        @click="submitQuery()"
+      >
+        <font-awesome-icon
+          :class="{'mt-1': showBothButtons}"
+          :icon="['fas', 'search']"
+        />
+      </b-button>
+      <input
+        type="submit"
+        hidden
+      >
+    </b-input-group-append>
+  </b-input-group>
+</template>
+
+<script>
+export default {
+  name: 'CInputSearch',
+
+  props: {
+    placeholder: {
+      type: String,
+      default: '',
+    },
+
+    disabled: {
+      type: Boolean,
+    },
+
+    showBothButtons: {
+      type: Boolean,
+    },
+
+    autocomplete: {
+      type: String,
+      default: 'on',
+    },
+
+    debounce: {
+      type: Number,
+      default: 0,
+    },
+  },
+
+  data () {
+    return {
+      value: '',
```

Comment:
The value should be a prop like we discussed here https://www.digitalocean.com/community/tutorials/how-to-add-v-model-support-to-custom-vue-js-components